### PR TITLE
Rolling window stride tricks

### PR DESCRIPTION
**Pull request by ssanderson**

### CODE REVIEW

File: zipline/utils/numpy_utils.py [hunk 1]
```diff
@@ -148,6 +148,52 @@ def repeat_last_axis(array, count):
...
+    ----------
+    array : np.ndarray
+        The base array.
+    length : int
+        Length of the synthetic first axis to generate.
+
+    Returns
+    -------
+    out : np.ndarray
+    """
```
Comment (llllllllll) [inline zipline/utils/numpy_utils.py]: can you include an example in the docstring. This is sort of hard to visualize

**Comment by richafrank** [comment]
@ssanderson What's next for this?  Looks like a nice utility!

**Comment by ssanderson** [comment]
@richafrank the build failure here is just a doctest, so probably easy enough to fix up and merge.  I haven't had a pressing need for this yet though, so I haven't bothered.  This might actually be a more efficient way to implement AdjustedArray though.

**Comment by richafrank** [comment]
Just pushed a potential fix.

**Comment by ssanderson** [comment]
@richafrank any reason not to push the button here?

**Comment by richafrank** [comment]
:+1: from me

**Comment by ssanderson** [comment]
button=pushed

**Pull request merged**